package.json

```diff
@@ -38,7 +38,7 @@
     "should": "^9.0.2"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
```

Comment:
This should be a separate PR, probably to be merged before this one, to better signal in the changelog that we're dropping support for node 4, which will be a breaking change.